CMakeLists.txt

```diff
@@ -363,7 +363,7 @@ target_compile_options(dunecommon INTERFACE ${dflags})
 target_compile_definitions(dunecommon INTERFACE DUNE_COMMON_VERSION_MAJOR=${DUNE_COMMON_VERSION_MAJOR})
 target_compile_definitions(dunecommon INTERFACE DUNE_COMMON_VERSION_MINOR=${DUNE_COMMON_VERSION_MINOR})
 target_compile_definitions(dunecommon INTERFACE DUNE_COMMON_VERSION_REVISION=${DUNE_COMMON_VERSION_REVISION})
-if(Boost_UNIT_TEST_FRAMEWORK_FOUND)
+if(Boost_UNIT_TEST_FRAMEWORK_FOUND AND BUILD_EXAMPLES)
```

Comment:
I think we should distinguish between examples and tests here. With this tests would only be built if BUILD_EXAMPLES is ON.

Reply:
hrm, indeed.

Reply:
or well, they would only be linked to dune-common. anyways fixed.